src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/action/stop/StopTransformRequest.kt

```diff
@@ -0,0 +1,31 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.stop
+
+import org.elasticsearch.action.ActionRequestValidationException
+import org.elasticsearch.action.ValidateActions.addValidationError
+import org.elasticsearch.action.update.UpdateRequest
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import java.io.IOException
+
+class StopTransformRequest : UpdateRequest {
```

Comment:
nit: same as start can have inline constructor

Reply:
See notes for start